src/clarity-angular/data/datagrid/built-in/filters/datagrid-string-filter.ts

```diff
@@ -68,7 +68,7 @@ export class DatagridStringFilter extends DatagridFilterRegistrar<DatagridString
                 // We need the timeout because at the time this executes, the input isn't
                 // displayed yet.
                 setTimeout(() => {
-                    this.renderer.invokeElementMethod(this.input.nativeElement, "focus");
+                    this.input.nativeElement.focus();
```

Comment:
Thank you, @youdz for this kind and detail explanation. I just checked the angular issue here, https://github.com/angular/angular/issues/13818